client/src/components/ShareButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ShareIcon } from '@heroicons/react/24/solid'
+import '../styles/ShareButton.css';
+
+type Props = {
+    hasShared: boolean,
+    clickHandler:React.MouseEventHandler<HTMLDivElement>
+};
+
+
+const ShareButton = ( { hasShared, clickHandler }: Props ) => {
+
+    return (
+        <div>
+            <div className='share__button' onClick={clickHandler}>
+                <ShareIcon className="share__button solid"/>
```

Comment:
To be consistent with the other buttons, could you please change the className here on line 16 from:
`share__button solid`
to:
`product__icon share__icon solid` ?

And you can remove the `<div>` tags from lines 18 and 20; the other buttons don't have these and it's best if all of the buttons follow the same format.

Reply:
@SirrorsMoore1975 I noticed you made the change to this class, so thank you, but it seems to have a typo. It is currently
`product__button share__icon solid`
but it should be
`product__icon share__icon solid`